MdeModulePkg/Core/Pei/Memory/MemoryServices.c

```diff
@@ -650,6 +652,28 @@ PeiAllocatePages (
     FreeMemoryBottom = &(Hob.HandoffInformationTable->EfiFreeMemoryBottom);
   }
 
+  // MU_CHANGE START
+  MemBucketHob = GetFirstGuidHob (&gMemoryBucketInformationGuid);

```

Comment:
Should this handle the case where the hob doesn't exist (i.e. GetFirstGuidHob returns NULL)?

Reply:
It's checked in the SyncMemoryBuckets function that it's passed into.